charts/microgateway/README.md.gotmpl

```diff
@@ -1,153 +1,365 @@
+```console
+helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml
+```
+
+:point_up: **YAML indentation**:<br>
+YAML is very strict with indetation. To ensure that the YAML file itself is correct, check it's content with a YAML validator (e.g. [YAML Lint](http://www.yamllint.com/)).
 
-To install the chart with the release name `waf`:
+## Dependencies
+
+The Airlock Microgateway Helm chart has the following optional dependencies, which can be enabled for an easy start.
+
+{{ template "chart.requirementsTable" . }}
+
+### Redis
+
+In case that sessionhandling is enabled on Airlock Microgateway, a Redis service needs to be available. The Airlock Microgateway Helm chart has two options:
+* To deploy the dependend Redis service, adapt the Helm chart configuration as shown below:
+  custom-values.yaml
+  ```
+  redis:
+    enabled: true
+  ```
+* To use an existing Redis service, adapt the Helm chart configuration as shown below:
+  custom-values.yaml
+  ```
+  config:
+    redisService:
+      - <REDIS-SERVICE1>:<PORT>
+      - <REDIS-SERVICE2>:<PORT>
+  redis:
+    enabled: false
+  ```
+
+Finally, apply the Helm chart configuration file with `-f` parameter.
 
 ```console
-helm upgrade -i waf airlock/microgateway
+helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml
 ```
 
-## Uninstalling the Chart
+:information_source: **Possible settings**:<br>
+Please refer to the [Redis Helm chart](https://hub.helm.sh/charts/bitnami/redis) to see all possible parameters of the Redis Helm chart.
+
+:warning: **Adjustments of the default settings**:<br>
+Please note that the dependend Redis service has been tested with the settings this Helm chart is delivered. Adjusting those settings can cause issues.
 
-To uninstall the chart with the release name `waf`:
+### Echo-Server
+
+For the first deployment it could be very useful to have a backend service processing requests. For this purpose the dependend echo-server can be deployed by doing the following:
+  custom-values.yaml
+  ```
+  echo-server:
+    enabled: true
+  ```
+
+Finally, apply the Helm chart configuration file with `-f` parameter.
 
 ```console
-helm uninstall waf
+helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml
 ```
 
-## <a name="dsl-configuration"></a> DSL Configuration
-With the Helm Chart you have three different possibilities to configure the DSL of the Microgateway. 
-Depending on the environment and use-case, another option may be the best and easiest choice for the implementation. 
+:information_source: **Possible settings**:<br>
+Please refer to the [Echo-Server Helm chart](https://ealenn.github.io/Echo-Server/pages/helm.html) to see all possible parameters of the Echo-Server Helm chart.
+
+## DSL configuration
+
+The Helm chart provides three different possibilities to configure the Microgateway DSL.
+Depending on the environment and use-case, another option might suit better and is easier to implement it.
```

Comment:
remove it.

Reply:
A heading without a short sentence is usually quite bad. Do you want another sentence there?

Reply:
I would remove the word it. at the end of the sentence
is easier to implement it. -->
is easier to implement.

Reply:
done